### PR TITLE
feat(Textarea): QUV-1099 New Textarea case with character counter

### PR DESCRIPTION
**Pull request by carlosnumber9**
This PR adds a new use case for the Textarea component, where a character counter is shown in its helper allowing the user to write under a limit of characters. Additionally, a change was made under the Helper component to not render a Paragraph when the value passed to helper prop is not a simple string.

### CODE REVIEW

**Comment by carlosnumber9** [comment]
Check this screen recording where it is possible to see the use case:

https://github.com/DevoInc/genesys-ui/assets/16353980/4159382d-bac9-4795-a12f-1f4a9ba4cc84

**Review by soslayando — APPROVED** [review]

File: packages/core/src/components/Helper/Helper.tsx [hunk 1]
```diff
@@ -53,12 +53,16 @@ export const Helper: React.FC<HelperProps> = ({
           />
         </Flex>
       )}
-      <Typography.Paragraph
-        colorScheme={status === 'error' ? 'error' : 'weak'}
-        size={size}
-      >
-        {message}
-      </Typography.Paragraph>
+      {React.isValidElement(message) ? (
```
[the remark this answers is not in the document]
Comment (carlosnumber9) [reply packages/core/src/components/Helper/Helper.tsx:64]: I think I'd rather choose the use of `isValidElement` function, as it belongs to the main library of the project (**React**) and I find it equally useful. However, I can make the change if we are using this option in similar situations inside of the project.

File: packages/form/src/components/Textarea/Textarea.stories.tsx [hunk 2]
```diff
@@ -34,3 +35,45 @@ export default meta;
...
+        setCounter(event.target.value.length);
+      };
+
+      return (
+        <Textarea
+          {...args}
+          onChange={onChange}
+          helper={
+            <HFlex justifyContent={'space-between'} width={'100%'}>
+              <Flex.Item alignSelf={'flex-start'}>
```
[the remark this answers is not in the document]
Comment (carlosnumber9) [reply packages/form/src/components/Textarea/Textarea.stories.tsx:59]: When I remove them, the character counter moves down when the error message displays.


https://github.com/DevoInc/genesys-ui/assets/16353980/4af9b805-3d07-46cf-816e-5eb8a2e51920

File: packages/core/src/components/Helper/Helper.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (soslayando) [reply packages/core/src/components/Helper/Helper.tsx:64]: Ok

File: packages/form/src/components/Textarea/Textarea.stories.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (carlosnumber9) [reply packages/form/src/components/Textarea/Textarea.stories.tsx]: Fixed! Avoided using Flex.Item just by adding the `alignItems` prop for HFlex

**Pull request merged**